src/main/java/org/ilgcc/app/inputs/Gcc.java

```diff
@@ -89,7 +89,7 @@ public class Gcc extends FlowInputs {
     private String streetAddress;
     private String city;
     private String state;
-    @Pattern(regexp = "^\\d{5,}$", message = "{errors.invalid-zipcode}")
+    @Pattern(regexp = "^[0-9]{5}(?:-[0-9]{4})?$", message = "{errors.invalid-zipcode}")
```

Comment:
This might be more simplified; 
```suggestion
    @Pattern(regexp = "^\d{5}(?:-\d{4})?$", message = "{errors.invalid-zipcode}")
```

Reply:
Ok, that [makes sense](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html#predef), changing

Reply:
@analoo, incorporated this change.